README.md

```diff
@@ -64,12 +64,14 @@ account in `security.json` is user `solr` using the password `SolrRocks`.
 
 To access the Solr container from DDEV's web container, use  `http://solr:8983`.
 
+More at [Drupal at your fingertips: Solr and DDEV](https://www.drupalatyourfingertips.com/development#solr-and-ddev).
+
+
+
 ## Create a collection
 
-It is recommended to use Solr's API to manage your collections (and cores).
-You could use the [Solr command line client](#solr-command-line-client), or the
-Solr API via curl or any http client.
-Or use Solr's admin UI.
+If you are using the Drupal Search API module, you don't need to do these steps, otherwise use Solr's API to manage your collections (and cores).
```

Comment:
Does this belong in the Drupal section?

Reply:
Maybe - I put it there in case people get hung up starting to create a collection without realizing that they don't need this step for Drupal.  I can remove it if you think it is best.

Reply:
Sorry I'm slow reviewing; @mkalkbrenner will be here before long too :)

Reply:
It's not like you are busy or anything

Reply:
Removed